sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewView.groovy

```diff
@@ -283,4 +297,35 @@ class ProjectOverviewView extends VerticalLayout{
             }
         }
     }
+
+    /**
+     * Determines the state of the current status. Is it in progress or did it complete already
+     * @param samplesInStatus the count for the specific status
+     * @param relativeCount
+     */
+    private static State determineCompleteness(RelativeCount relativeCount) {
+        int samplesInStatus = relativeCount.getValue()
+        int totalSamples = relativeCount.getTotal()
+        if (samplesInStatus == totalSamples) {
+            return State.COMPLETED
+        } else if (samplesInStatus < totalSamples) {
+            return State.IN_PROGRESS
+        } else {
+            //unexpected!!
```

Comment:
NIT: I think this is shown by the IllegalStateException
```suggestion
```

Reply:
I think this is still a valid comment since we throw an invalid state exception